### PR TITLE
change cherrypy.wsgiserver to cheroot.server

### PR DESCRIPTION
**Pull request by raven38**
change cherrypy.wsigiserver because delete cherrypy.wsgiserver in cherrypy. 
see https://github.com/cherrypy/cherrypy/commit/929695cd3efedcb3e416b84d3369b33f1ee96f4f.

### CODE REVIEW

**Comment by codypiersall** [comment]
I just ran in to this problem too and was about to provide the same patch.  It _might_ be worth putting a `try: except`  block around the import so that ws4py will still work with older cherrypy versions, but that's up to the maintainer.

**Comment by binary1230** [comment]
hey, just ran into this too our our cherrypy powered app, would love to see this fix go in

**Comment by raven38** [comment]
I change cherrypyserver.py to support older cheerypy versions.

**Comment by binary1230** [comment]
ohhh right.  that change totally makes sense, good call @codypiersall  thanks @raven38

File: ws4py/server/cherrypyserver.py [hunk 1]
```diff
@@ -69,7 +69,10 @@ def ws(self):
 import cherrypy
 from cherrypy import Tool
 from cherrypy.process import plugins
-from cherrypy.wsgiserver import HTTPConnection, HTTPRequest, KnownLengthRFile
+try:
+    from cherrypy.wsgiserver import HTTPConnection, HTTPRequest, KnownLengthRFile
+except ImportError:
+    from cheroot.server import HTTPConnection, HTTPRequest, KnownLengthRFile
```
Comment (webknjaz) [inline ws4py/server/cherrypyserver.py]: I would swap these imports.

**Comment by webknjaz** [comment]
Ref #205

File: ws4py/server/cherrypyserver.py (hunk 1 above)
Comment (raven38) [inline ws4py/server/cherrypyserver.py]: Thank your review to my PR.
That's better. I fix as you say.

**Comment by webknjaz** [comment]
@Lawouach

**Comment by binary1230** [comment]
thanks for fixing this everyone! awesome stuff

**Comment by raven38** [comment]
Thank you for merging my PR. I mind build failed.　Are you all right?

**Comment by webknjaz** [comment]
@raven38 there's nothing to worry about as long as master branch test are green

**Pull request merged**